sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpClient.cs

```diff
@@ -175,6 +175,29 @@ internal class AmqpClient : TransportClient
             );
         }
 
+        /// <summary>
+        ///   Creates a rule manager strongly aligned with the active protocol and transport,
+        ///   responsible for adding, removing and getting rules from the Service Bus subscription.
+        /// </summary>
+        ///
+        /// <param name="entityPath">The name of the Service Bus entity to which the rule manager is bound.</param>
+        /// <param name="retryPolicy">The policy which governs retry behavior and try timeouts.</param>
+        ///
+        /// <returns>A <see cref="TransportRuleManager"/> configured in the requested manner.</returns>
+        public override TransportRuleManager CreateRuleManager(
+            string entityPath,
```

Comment:
The entity is always a subscription. Should it be explicitly called the subscription path?

Reply:
We take topicName and subscriptionName as parameters when we [create rule manager](https://github.com/ShivangiReja/azure-sdk-for-net/blob/AddRuleManager/sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs#L566) and concatenate both by calling `EntityNameFormatter.FormatSubscriptionPath(topicName, subscriptionName))` in the ServiceBusClient. When we concatenate topicName and subscriptionName we call it `entityPath` and pass it to the amqp layer.

Reply:
Updated `entityPath` to `subscriptionPath`. Thanks for the suggestion.